contrib/debian/changelog

```diff
@@ -178,7 +178,7 @@ bitcoin (0.5.0~rc1-natty1) natty; urgency=low
 
   * Add test_bitcoin to build test
   * Fix clean
-  * Remove uneccessary build-dependancies
+  * Remove unnecessary build-dependancies
```

Comment:
dependancies, huh :-)
This is just a backport, but I wonder why the other word spelling wasn't fixed.